ui/main/src/app/modules/cards/components/detail/detail.component.spec.ts

```diff
@@ -102,7 +100,7 @@ describe('DetailComponent', () => {
         fixture.detectChanges();
         expect(component).toBeTruthy();
     });
-    it('should load template with script', ()=>{
+    fit('should load template with script', ()=>{
```

Comment:
You miss this focus on this test.

Reply:
fit remove and correction to another test has been done